docs/changelog.rst

```diff
@@ -6,14 +6,18 @@ Version 5.3.0
 
 KSQL 5.3.0 includes new features, including:
 
+* Drop the requirement that ``CREATE TABLE`` statements must have a ``KEY`` set in their ``WITH`` clause.
+  This is now an optional optimisation to avoid unnecessary repartition steps.
```

Comment:
Is it `optimisation` or `optimization`? Seems the first one is a British word, but you're using the second one in the `syntax-reference.rst` section.

Reply:
It took me a while to realize (realise?) that artefact is a legitimate spelling of what I've always seen as "artifact"

Reply:
I'm British ;)